KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutViewController.swift

```diff
@@ -81,6 +81,10 @@ final class KarhooCheckoutViewController: UIViewController, CheckoutView {
         footerView.translatesAutoresizingMaskIntoConstraints = false
         footerView.accessibilityIdentifier = "footer_view"
         footerView.backgroundColor = .white
+        footerView.layer.shadowColor = KarhooUI.colors.textLabel.cgColor
```

Comment:
Use `KarhooUI.colors.black.cgColor` and decrease shadow opacity to make it transparent.